src/app/tabs/compile-tab.js

```diff
@@ -317,3 +322,141 @@ function detailsHelpSection () {
 }
 
 module.exports = compileTab
+
+const css = csjs`
```

Comment:
well I think it would really make sense to keep them separated

Reply:
see https://github.com/ethereum/remix-ide/pull/1235#discussion_r180827235

Reply:
ok